vmdb/spec/models/ems_refresh/refreshers/openstack_refresher_rhos_grizzly_spec.rb

```diff
@@ -67,7 +67,7 @@ def assert_table_counts
     SystemService.count.should       == 0
 
     Relationship.count.should        == 13
-    MiqQueue.count.should            == 19
+    MiqQueue.count.should            == 20
```

Comment:
Yeah, @blomquisg, I agree, it will bite us though so if we knew what the intent was here, we could eliminate the brittleness and test something we really care about.  As it is now, I can't guess what MiqQueue.count could mean.

I'm not against merging but clearly this will hit us again.